app/marko/src/server/config/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+import path from 'path';
+import webpack from 'webpack';
+import Dotenv from 'dotenv-webpack';
+import CaseSensitivePathsPlugin from 'case-sensitive-paths-webpack-plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import {
+  managerPath,
+  getPreviewHeadHtml,
+  getManagerHeadHtml,
+  indexHtmlPath,
+  iframeHtmlPath,
+} from '@storybook/core/server';
+
+import { includePaths, excludePaths, loadEnv, nodePaths } from './utils';
+import babelLoaderConfig from './babel';
+import { version } from '../../../package.json';
+
+export default function(configDir, quiet) {
+  const config = {
+    mode: 'development',
+    devtool: 'cheap-module-source-map',
+    entry: {
+      manager: [require.resolve('./polyfills'), managerPath],
+      preview: [
+        require.resolve('./polyfills'),
+        require.resolve('./globals'),
+        `${require.resolve('webpack-hot-middleware/client')}?reload=true`,
+      ],
+    },
+    output: {
+      path: path.join(__dirname, 'dist'),
+      filename: 'static/[name].bundle.js',
+      publicPath: '/',
+    },
+    plugins: [
+      new HtmlWebpackPlugin({
+        filename: 'index.html',
+        chunks: ['manager'],
+        chunksSortMode: 'none',
+        data: {
+          managerHead: getManagerHeadHtml(configDir),
+          version,
+        },
+        template: indexHtmlPath,
+      }),
+      new HtmlWebpackPlugin({
+        filename: 'iframe.html',
+        excludeChunks: ['manager'],
+        chunksSortMode: 'none',
+        data: {
+          previewHead: getPreviewHeadHtml(configDir),
+        },
+        template: iframeHtmlPath,
+      }),
+      new webpack.DefinePlugin(loadEnv()),
+      new webpack.HotModuleReplacementPlugin(),
+      new CaseSensitivePathsPlugin(),
+      quiet ? null : new webpack.ProgressPlugin(),
+      new Dotenv({ silent: true }),
+    ].filter(Boolean),
+    module: {
+      rules: [
+        {
+          test: /\.marko$/,
+          loader: require.resolve('marko-loader'),
```

Comment:
you need to add it as a direct dependency of `@storybook/marko`

Reply:
Should we remove it from the root package.json?

Reply:
As you wish

Reply:
ok. will just leave it there.